packages/transport/src/utils/bridgeApiCall.ts

```diff
@@ -4,6 +4,9 @@ import { success, error, unknownError } from './result';
 
 import * as ERRORS from '../errors';
 
+// import { PROTOCOL_MALFORMED  } from '@trezor/protocol';
+export const PROTOCOL_MALFORMED = 'Malformed protocol format' as const;
```

Comment:
Why this and not that?

Reply:
fixed 57a36cdcc9b162760a6ae1a448ddd3f3132fa570